### PR TITLE
Add robot test for "Enable record type validation"

### PR DESCRIPTION
**Pull request by dashattuck**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

**Comment by dashattuck** [comment]
@jstvz is restarting the build.

**Comment by dashattuck** [comment]
@xinnanlisf  - this is ready for review

File: tests/browser/heda_settings_affiliations_edit_mode.robot [hunk 1]
```diff
@@ -15,17 +15,16 @@ Validate Edit Mode For Affiliations, Settings
...
 
+    #Save the changes
+    Select Checkbox                 //label[@class='slds-checkbox']//input[@type='checkbox']
 
-#    #Save the changes
-#    Select Checkbox                 //label[@class='slds-checkbox']//input[@type='checkbox']
-#    Click Element                   //div[@class='slds-button-group']//span[contains(text(), 'Save')]
-    
+    # Record Type Validation
+    Click Element                   //input[contains(@class,'store-errors')]/parent::label
```
Comment (xinnanlisf) [inline tests/browser/heda_settings_affiliations_edit_mode.robot]: @dashattuck curious to hear what this line does. Does this line or "Select Checkbox" check the checkbox?

Comment (dashattuck) [reply tests/browser/heda_settings_affiliations_edit_mode.robot]: This places a check in the checkbox for Enable Record Type Validation.  It subsequently gets saved.

Comment (xinnanlisf) [reply tests/browser/heda_settings_affiliations_edit_mode.robot]: @dashattuck yes that makes sense. I am just a little surprised to see it is finding the "store-errors" class.

Comment (xinnanlisf) [reply tests/browser/heda_settings_affiliations_edit_mode.robot]: @dashattuck feel free to change the class name reference here.

**Comment by dashattuck** [comment]
@xinnanlisf  - I made the requested change to the robot file.  Let me know if we need anything else.

**Comment by xinnanlisf** [comment]
@dashattuck looks good, feel free to merge it. Cheers!

**Comment by dashattuck** [comment]
Thanks Xinnan!

**Pull request merged**